github_test.go

```diff
@@ -9,22 +9,27 @@ import (
 
 func TestGetListOfReleasesFromGitHubRepo(t *testing.T) {
 	t.Parallel()
+  testInst := GitHubInstance{
+    BaseUrl: "github.com",
+    ApiUrl: "api.github.com",
+  }
```

Comment:
Indentation? Please check your editor settings.

Reply:
Yep. I thought I had `vim-go` installed, but had `vim-polyglot` instead which wasn't cutting it on it's own.